.github/workflows/cd-snapshot.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - name: Set up Node
         uses: actions/setup-node@v2
         with:
-          node-version: '16'
+          node-version: '20'
```

Comment:
Not sure if this has anything to do with the WIP issue (which seems to be resolved now). I checked but there were no GitHub incidents about Actions today / yesterday, wonder what could have happened. In any case seems like functionality is back up after what ET tried?

Reply:
No this is to fix the snapshot action failing. It's failing on every other open PR too.